sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusManagementClient.java

```diff
@@ -0,0 +1,202 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.messaging.servicebus;
+
+import com.azure.core.annotation.ReturnType;
+import com.azure.core.annotation.ServiceClient;
+import com.azure.core.annotation.ServiceMethod;
+import com.azure.core.http.rest.PagedFlux;
+import com.azure.core.http.rest.PagedIterable;
+import com.azure.core.http.rest.Response;
+import com.azure.core.util.Context;
+import com.azure.messaging.servicebus.models.QueueDescription;
+import com.azure.messaging.servicebus.models.QueueRuntimeInfo;
+
+import java.util.Objects;
+
+/**
+ * A <b>synchronous</b> client for managing a Service Bus namespace.
+ *
+ * @see ServiceBusManagementAsyncClient ServiceBusManagementAsyncClient for an asynchronous client.
+ */
+@ServiceClient(builder = ServiceBusManagementClientBuilder.class)
+public final class ServiceBusManagementClient {
+    private final ServiceBusManagementAsyncClient asyncClient;
+
+    /**
+     * Creates a new instance with the given client.
+     *
+     * @param asyncClient Asynchronous client to perform management calls through.
+     */
+    ServiceBusManagementClient(ServiceBusManagementAsyncClient asyncClient) {
+        this.asyncClient = Objects.requireNonNull(asyncClient, "'asyncClient' cannot be null.");
+    }
+
+    /**
+     * Creates a queue the {@link QueueDescription}.
+     *
+     * @param queue Information about the queue to create.
+     *
+     * @return The created queue.
+     * @throws NullPointerException if {@code queue} is null.
+     * @throws IllegalArgumentException if {@link QueueDescription#getName() queue.getName()} is null or an empty
+     *     string.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public QueueDescription createQueue(QueueDescription queue) {
+        return asyncClient.createQueue(queue).block();
```

Comment:
`queueDescription`  as  arg name will be more appropriate ?

Reply:
Went with .NET's parameter name.